extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeChannelExtractorTest.java

```diff
@@ -48,6 +61,91 @@ public void nonExistentFetch() throws Exception {
                     YouTube.getChannelExtractor("https://www.youtube.com/channel/DOESNT-EXIST");
             extractor.fetchPage();
         }
+
+        @Test(expected = AccountTerminatedException.class)
+        public void accountTerminatedTOSFetch() throws Exception {
+            final ChannelExtractor extractor =
+                    YouTube.getChannelExtractor("https://www.youtube.com/channel/UCTGjY2I-ZUGnwVoWAGRd7XQ");
+            try {
+                extractor.fetchPage();
+            } catch (AccountTerminatedException e) {
+                assertEquals(e.getMessage(),
+                        "This account has been terminated for a violation of YouTube's Terms of Service.");
```

Comment:
I would remove the assertions for the error message. They are bound to fail very soon, if they change the wording. Only usecase i can think of for them, is directly displaying them to the end-user in which case, the exact wording does not change the correctness of the implementation.